src/sentry/static/sentry/app/components/charts/lineChart.jsx

```diff
@@ -16,12 +16,12 @@ export default class LineChart extends React.Component {
     return (
       <BaseChart
         {...props}
-        series={series.map(({seriesName, data, ...options}) => {
+        series={series.map(({seriesName, data, dataArray, ...options}) => {
           return LineSeries({
             ...seriesOptions,
             ...options,
             name: seriesName,
-            data: data.map(({value, name}) => [name, value]),
+            data: dataArray || data.map(({value, name}) => [name, value]),
```

Comment:
This is to avoid an extra iteration of the data array if it's already in `[name, value]` format.